frontend/package.json

```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "vite": "^4.1.2"
+    "vite": "^4.1.2",
+    "alpinejs": "^3.11.1"
```

Comment:
Isn't this actually a library that should be added to the `dependencies` not `devDependencies`? I mean this will be definitely used in the production and has nothing to do with build / linters / tests